### PR TITLE
Implement getWalletBalance 

### LINKED ISSUE #590: Expose functions for getting wallet balance
There's an incomplete attempt: https://github.com/Plutonomicon/cardano-transaction-lib/pull/485/files

We need to extract useful parts from there.

### PR DESCRIPTION
**Pull request by klntsky**
Closes #590 

### Pre-review checklist

- [x] All code has been formatted using our config (`make format` for Purescript, `nixpkgs-fmt` for Nix)
- [x] All Purescript imports are explicit, including constructors
- [ ] **All** existing examples have been run locally against a fully-synced testnet node
- [x] The integration and unit tests have been run (`npm run test`) locally
- [x] The changelog has been updated under the `## Unreleased` header, using the appropriate sub-headings (`### Added`, `### Removed`, `### Fixed`)

I tried Gero & Nami examples. It **should not** work with KeyWallet until #617 is implemented. After that it **should** work without extra changes.

### CODE REVIEW

**Review by firefrorefiddle — APPROVED** [review]
Looks good!

File: src/Deserialization/FromBytes.purs [hunk 1]
```diff
@@ -120,3 +124,6 @@ foreign import _fromBytesMint
 
 foreign import _fromBytesVRFKeyHash
   :: forall r. ErrorFfiHelper r -> ByteArray -> E r VRFKeyHash
+
+foreign import _fromBytesValue
+  :: forall r. ErrorFfiHelper r -> ByteArray -> E r Value
```
Comment (vvtran) [inline src/Deserialization/FromBytes.purs:129]: Could you `r :: Row Type` this and everything around it, including `eh` please?

**Review by vvtran — COMMENTED** [review]
There's an issue with the return type in `Contract` version of `getWalletBalance`

**Review by ngua — APPROVED** [review]
LGTM, just need to fix the merge conflict in the changelog

**Pull request merged**